content/reference/endpoints/rental_cancelation_policies.md

```diff
@@ -0,0 +1,43 @@
+# Rental Cancelation Policies
+
+1. TOC
+{:toc}
+
+### Parameters
+<ul class="nav nav-pills" role="tablist">
+  <li class="disabled"><a>OAuth Scopes:</a></li>
+  <li class="active"><a href="#public" role="tab" data-toggle="pill">public</a></li>
+</ul>
+<div class="tab-content" markdown="1">
+  <div class="tab-pane active" id="public" markdown="1">
+Name             | Type    | Read/Write | Description
+-----------------|---------|------------|------------
+id               | Integer | Read       | Rental cancelation policy's id.
+-----------------|---------|------------|------------
+body_translations | [Object](/reference/enums#formats)     | Read       | Rental cancelation policy's body, list of locales described in [enums section](/reference/enums#locales).
```

Comment:
missing `body` attribute as well - seems to be body translation in default locale